tools/src/tester/StoryOutputs.ts

```diff
@@ -31,7 +31,8 @@ export class StoryOutputs {
   }
 
   get_output_value (chapter_id: string, output_name: string): any {
-    return this.outputs[chapter_id].get_output(output_name)
+    const output = this.outputs[chapter_id]
+    return output !== undefined ? output.get_output(output_name) : undefined
```

Comment:
You should be able to just use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) here:
```suggestion
    return this.outputs[chapter_id]?.get_output(output_name)
```

Reply:
You can't because this is a `Record<string, ChapterOutput>` and the type system is not smart enough to know that the value can be `undefined`.